src/Swashbuckle.AspNetCore.SwaggerGen/Generator/OpenApiAnyFactory.cs

```diff
@@ -25,14 +25,14 @@ public static bool TryCreateFor(OpenApiSchema schema, object value, out IOpenApi
             else if (schema.Type == "number" && schema.Format == "double" && TryCast(value, out double doubleValue))
                 openApiAny = new OpenApiDouble(doubleValue);
 
-            else if (schema.Type == "string" && value.GetType().IsEnum)
+            else if (schema.Type == "string" && value !=null && value.GetType().IsEnum)
```

Comment:
I think it's better to format it, add whitespace before null

Reply:
Done :)